src/server/ua_nodes.c

```diff
@@ -546,6 +546,9 @@ UA_Node_copy(const UA_Node *src, UA_Node *dst) {
     UA_StatusCode retval = UA_NodeId_copy(&srchead->nodeId, &dsthead->nodeId);
     retval |= UA_QualifiedName_copy(&srchead->browseName, &dsthead->browseName);
 
+    /* Copy the RolePermissions */
+    memcpy(dsthead->rolePermissions, srchead->rolePermissions, sizeof(UA_RoleSet) * 16);
```

Comment:
Why not using sizeof(dsthead->rolePermissions) for the size or the named constant "UA_ROLEPERMISSIONS_COUNT" instead of "16" here?

Reply:
Good find. Will change that.